app/src/main/java/ch/hikemate/app/ui/profile/EditProfileScreen.kt

```diff
@@ -182,6 +190,15 @@ fun EditProfileScreen(
                         profile.joinedDate)
                 profileViewModel.updateProfile(profile = savedProfile!!)
                 isLoading = true
+                // Add a timeout in case the user is offline. This would still
+                // update the profile as Firebase Firestore syncs when the user
+                // goes online.
+                coroutineScope.launch {
+                  delay(EditProfileScreen.UPDATE_TIMEOUT)
+                  isLoading = false
+                  savedProfile = null
+                  navigationActions.goBack()
+                }
```

Comment:
Warning: I'm not sure of how this works as a Timeout? Imo, this will run no matter what and introduces a new bug in case the user reedit during the 5s timeout?

Reply:
Same answer as first one

Reply:
Ooooh I see! Thanks!